docs/build_api_docs.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 The Cirq Authors. All Rights Reserved.
```

Comment:
Can please you move this file into `dev_tools/docs/` ?

Reply:
Done.